xml/depl_nodes.xml

```diff
@@ -552,14 +549,14 @@
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term><guimenu>HAProxy: Public name for public virtual IP</guimenu>
+    <term><guimenu>HAProxy: Public name for public virtual IP address</guimenu>
```

Comment:
Hm, as this is within a guimenu element, the string is called like that in the SW. Unless we can convince the developers to change the string, we should keep it (at least, within the guimenu element).